### PR TITLE
feat(chore): added types for input, input-switch, label and menu [KHCP-6344]

### PR DESCRIPTION
**Pull request by LukeSwierlik**
Task in Jira: https://konghq.atlassian.net/browse/KHCP-6344

# Summary
- **Feat** - Added types for `Input`.
- **Feat** - Added types for `Input-Switch`.
- **Feat** - Added types for `Label`.
- **Feat** - Added types for `Menu`.

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Review by adamdehaven — DISMISSED** [review]
lgtm

File: src/types/input.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export type Size = 'small' | 'medium' | 'large'
+export type SizeRecord = Record<Size, Size>
+
+export type IconPosition = 'start' | 'end'
+export type IconPositionRecord = Record<IconPosition, IconPosition>
+
+export interface LabelAttributes {
+  help: string;
+  info: string;
```
Comment (adamdehaven) [inline src/types/input.ts:11]: Can you remove the semi-colons? (and search the other type files for semi-colons and commas and remove if not needed)?

**Review by adamdehaven — APPROVED** [review]
lgtm

File: src/components/KInput/KInput.vue [hunk 2]
```diff
@@ -225,34 +246,38 @@ export default defineComponent({
 
     watch(value, (newVal, oldVal) => {
       if (newVal !== oldVal) {
-        handleInput({ target: { value: newVal } })
+        updateInputValue(String(newVal))
```
Comment (adamdehaven) [inline src/components/KInput/KInput.vue:249]: @sumimakito I'm wondering if this is where the KInput behavior inadvertently changed?

Comment (sumimakito) [reply src/components/KInput/KInput.vue:249]: @adamdehaven Yes. This should be the line that caused this change.

**Pull request merged**